ui/webui/src/app.jsx

```diff
@@ -33,15 +41,60 @@ export const Application = () => {
 
     useEffect(() => cockpit.file("/run/anaconda/bus.address").watch(setAddress), []);
 
+    const steps = [
+        {
+            id: 0,
+            name: "Installation language",
```

Comment:
Please translate the step names. You can do it like: 
_("Installation language")

Reply:
Added.